packages/not-ocamlfind/not-ocamlfind.0.07.02/opam

```diff
@@ -0,0 +1,35 @@
+opam-version: "2.0"
+version: "0.07.02"
+synopsis: "A small frontend for ocamlfind that adds a few useful commands"
+maintainer: "Chet Murthy <chetsky@gmail.com>"
+
+(* Gerd wrote most of this code; I just modified it (and probably
+introduced bugs.  This is to silence opam *)
+
+authors: "Chet Murthy <chetsky@gmail.com>"
+homepage: "https://github.com/chetmurthy/not-ocamlfind"
+bug-reports: "Chet Murthy <chetsky@gmail.com>"
+depends: [
+  "ocamlfind" {>= "1.8.0"}
+  "fmt" {>= "0.8.8"}
+  "rresult" {>= "0.6.0"}
+  "ocamlgraph" {>= "2.0.0"}
+  "conf-which"
+]
+depexts: [
+  [
+    "xdot"
+  ] {os-family = "debian"}
```

Comment:
I'm curious (not a blocker), why is it needed only on debian?

Reply:
there's function (to draw a graph of findlib packages starting from some initial set) that needs xdot.  I was too lazy to figure out what the right package would be on each distro (and at the time, I only knew the answer for debian/ubuntu).  Since it was
1. neither needed for the build
2. nor for testing,
3.  figured I wouldn't bother figuring out how to get it working on all the other distros, until somebody asked.

The useful part of not-ocamlfind is "reinstall-if-diff" and "preprocess"; this subcommand  (`package-graph`) is .... well, it's nice, but really, I don't expect anybody to install not-ocamlfind for that functionality.

Would you prefer that I figure out how to enable it on all the other platforms?

Reply:
Ideally yes, but it's ok for now given previous versions also did the same. Do it when you are bored and have some spare time.